tests/test_subdirectory.py

```diff
@@ -152,7 +152,21 @@ def test_update_subdirectory_from_root_path(tmp_path_factory):
     assert (dst / "subfolder" / "file1").read_text() == "version 2\nhello\na1\nbye\n"
 
 
-def test_new_version_uses_subdirectory(tmp_path_factory):
+@pytest.mark.parametrize(
+    "conflict, readme, expect_reject",
+    [
+        ("rej", "upstream version 2\n", True),
+        (
+            "inline",
+            "<<<<<<< modified\ndownstream version 1\n"
+            "=======\nupstream version 2\n>>>>>>> new upstream\n",
```

Comment:
It's weird that all other `\n` work out fine. 🤔 

```suggestion
            f"<<<<<<< modified\ndownstream version 1{os.linesep}"
            "=======\nupstream version 2\n>>>>>>> new upstream\n",
```